charts/prometheus/tests/prometheus-config-configmap_test.yaml

```diff
@@ -3,7 +3,28 @@ suite: Test prometheus-config-configmap
 templates:
   - prometheus-config-configmap.yaml
 tests:
-  - it: should work
+  - it: is the correct type of resource
     asserts:
       - isKind:
           of: ConfigMap
+  - it: contains exactly one document
+    asserts:
+      - hasDocuments:
+          count: 1
+  - it: does not include substrings that appear to mix up namespace and helm release name
+    release:
+      name: my-release
+      namespace: my-namespace
+    set:
+      global.blackboxExporterEnabled: true
+      global.veleroEnabled: true
+      global.prometheusPostgresExporterEnabled: true
+      global.nodeExporterEnabled: true
+      tcpProbe.enabled: true
+    asserts:
+      - notMatchRegex:
+          path: data.config
```

Comment:
this is the part that was not working with 'prometheus.yaml' key, including a dot in the key name